configuration.hpp

```diff
@@ -68,8 +68,10 @@ namespace irods {
             std::string maximum_delay_time{"30"};
             std::string job_limit{""};
             std::string delay_parameters{"<EF>60s DOUBLE UNTIL SUCCESS OR 5 TIMES</EF>"};
-            std::string urlTemplate{"http:/{}"};
-
+            std::string urlTemplate{"http:/{}"};  // Clients should aim not to use this index-embedded URL. It is no longer used by
+	                                          // MetaLnx as of version 2.5.0, nor its search plugin at:
+						  //    https://github.com/trel/file-and-metadata-search-plugin
```

Comment:
whitespace alignment

let's leave out the link/mention to the search plugin for now - it won't live there or be named that once we pull it into the irods namespace.

Reply:
ok